src/main/scala/mesosphere/marathon/core/launchqueue/impl/AppTaskLauncherActor.scala

```diff
@@ -321,8 +335,9 @@ private class AppTaskLauncherActor(
     sender() ! QueuedTaskCount(
       app,
       tasksLeftToLaunch = tasksToLaunch,
-      taskLaunchesInFlight = inFlightTaskLaunches.size,
-      tasksLaunchedOrRunning = tasksMap.size - inFlightTaskLaunches.size,
+      taskLaunchesInFlight = inFlightTaskOperations.size,
+      // don't count tasks that are not launched in the tasksMap
+      tasksLaunchedOrRunning = tasksMap.values.count(_.launched.isDefined) - inFlightTaskOperations.size,
```

Comment:
The original line before any ResidentTasks changes was
```
tasksLaunchedOrRunning = tasksMap.size - inFlightTaskLaunches.size
```
The scala doc states that `tasksLaunchedOrRunning` is _the number of tasks which are running or at least have been confirmed to be launched_ – so that was hard to understand from the beginning. The way I see it, we have
- `tasksLeftToLaunch` – not yet launched at all
- `taskLaunchesInFlight` – we created a TaskOp but that's not been accepted internally == not yet send to the scheduler via `acceptOffer()`
- `tasksLaunchedOrRunning` – either running or at least already processed via `acceptOffer()` – these are NOT in flight, therefore the inFlightTaskOperations.size is subtracted.

I think the naming here is really misleading and we should either make that crystal clear with good names and/or clarify with tests. Any suggestions?